routes/web.php

```diff
@@ -20,8 +20,27 @@
 Route::get('', 'HomeController@index')->name('index');
 Route::get('about', 'HomeController@aboutPage')->name('about');
 Route::post('kmlpolygonsearch', 'GazetteerController@searchFromKmlPolygon')->name('searchFromKmlPolygon'); //search from file
-Route::get('places/{id?}', 'GazetteerController@search')->name('places'); //shows places with optional id, if no id is given it uses all results before applying filters
-Route::get('search', 'GazetteerController@search')->name('search')->middleware('checkmaxpaging', 'cors');
+
+// Redirect from search to places
+Route::get('search', function (Request $request) {
+    $queryParameters = $request->query();  // Get all query parameters
+
+    $redirectURL = '/places';
+
+    // if ($id = $request->input('id')) {
+    //     $redirectURL .= '/' . $id;  //TODO  comfirm this
+    // }
+
+    // Build query string and append it to redirect URL
+    if (!empty($queryParameters)) {
+        $redirectURL .= '?' . http_build_query($queryParameters);
+    }
+
+    return redirect($redirectURL);
+});

```

Comment:
This seems to be unnecessarily complex. Can just create a simple redirect from `search` to `places`.